CHANGELOG.adoc

```diff
@@ -85,6 +85,37 @@ Additionally, `ErrorStatusCode` provides an `as_client_error` method that
 returns a `ClientErrorStatusCode` if the status code is a client error, or an
 error.
 
+* `HttpError` now contains an `Option<Box<http::HeaderMap>>` of headers to add
+to error responses constructed for the error.
++
+Code which constructs `HttpError` literals must now initialize this field. For
+example, this will no longer compile:
++
+```rust
+let err = dropshot::HttpError {
+    status_code: ErrorStatusCode::INTERNAL_SERVER_ERROR,
+    error_code: None
+    internal_message: "something bad happened".to_string(),
+    external_message: "Internal Error".to_string(),
+};
+```
++
+Instead, write:
++
+```rust
+let err = dropshot::HttpError {
+    status_code: ErrorStatusCode::INTERNAL_SERVER_ERROR,
+    error_code: None
+    internal_message: "something bad happened".to_string(),
+    external_message: "Internal Error".to_string(),
+    headers: None, // <-- Add this
+};
+```
```

Comment:
```suggestion
Code that constructs `HttpError` literals must now initialize this field.
```

Reply:
Hm, @davepacheco had previously expressed a preference for including examples of breaking changes whenever possible...though I agree that this one is pretty trivial. Happy to remove it.

Reply:
Removed it in a34b66c1a2c2293a23cfa4ac3f3bd54effffe053.